src/lib/imports.ts

```diff
@@ -1,68 +1,90 @@
-import { Database } from 'better-sqlite3'
-import { Static, Type as T } from '@sinclair/typebox'
-
-import { NullableSchema } from './utils'
+import type { Database } from 'better-sqlite3'
```

Comment:
This file used to have a bunch of runtime types which were used by the route. Now they're only compile-time types.

I also moved the documentation out of `API.md` into this file.